Provis.UnitTests/Core/Services/UserServiceTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Options;
+using Moq;
+using NUnit.Framework;
+using Provis.Core.DTO.UserDTO;
+using Provis.Core.Entities.InviteUserEntity;
+using Provis.Core.Entities.UserEntity;
+using Provis.Core.Exeptions;
+using Provis.Core.Helpers;
+using Provis.Core.Interfaces.Repositories;
+using Provis.Core.Interfaces.Services;
+using Provis.Core.Services;
+using Provis.UnitTests.Base;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace Provis.UnitTests.Core.Services
+{
+    [TestFixture]
```

Comment:
Let's use terar down and verify all mocks

Reply:
terar down method is added